cli/src/types.rs

```diff
@@ -416,9 +416,16 @@ pub struct PurlWithOrigin {
 pub struct UserGroup {
     pub created_at: DateTime<Utc>,
     pub last_modified: DateTime<Utc>,
+    pub group_id: Option<String>,
```

Comment:
https://github.com/phylum-dev/api/pull/1503 technically doesn't break things, it just removes the admin information. So technically we could wait until that PR is in prod and then remove the `Option` here.